draft-ietf-quic-multipath.md

```diff
@@ -1238,8 +1248,31 @@ Note that the RETIRE_CONNECTION_ID frame can only be used to retire
 connection IDs for the initial path with Path ID 0.
 
 As the MP_NEW_CONNECTION_ID frames applies the sequence number per path,
-the sequence number in the MP_RETIRE_CONNECTION_ID frame
-also needs to be considered in the context of the Path Identifier field.
+the sequence number in the MP_RETIRE_CONNECTION_ID frame is also per
+path. The MP_RETIRE_CONNECTION_ID frame retires the Connection ID with
+the specified path ID and sequence number.
+
+Receipt of an MP_RETIRE_CONNECTION_ID frame containing a sequence number
+greater than any previously sent to the peer for the specified path ID
+MUST be treated as a connection error of type PROTOCOL_VIOLATION.
+If the multipath extension is negotiated, receipt of
+a RETIRE_CONNECTION_ID frame containing a sequence number
+greater than any previously sent to the peer for path ID 0 MUST
+be treated as a connection error of type PROTOCOL_VIOLATION,
+regardless of how many path IDs have been sent for other paths.
+
+The sequence number specified in an MP_RETIRE_CONNECTION_ID frame
+MUST NOT refer to the Destination Connection ID field of the packet
+in which the frame is contained. The peer MAY treat this as a connection
+error of type PROTOCOL_VIOLATION. If the multipath extension is negotiated,
+the same error is detected if a RETIRE_CONNECTION_ID frame is received,
+the path ID of the Destination Connection ID field of the packet is 0,
+and the sequence number of that connection ID matches Sequence Number
+field of the RETIRE_CONNECTION_ID frame.
+
+These last two paragraphs change on
+the processing of RETIRE_CONNECTION_ID frames specified in
+{{Section 19.17 of QUIC-TRANSPORT}}.
```

Comment:
I don't think we should repeat all normative language from RFC9000 here. I would rather prefer something like:

```
The processing as described In {{Section 19.17 of QUIC-TRANSPORT}} applies for MP_RETIRE_CONNECTION_ID frames per path, while the processing of an RETIRE_CONNECTION_ID frame is only applied for Path ID 0.
```

Reply:
What changes is the processing of the plain RETIRE_CONNECTION_ID, and that is a deviation from RFC 9000. Implementations need not just implement the new MP_RETIRE_CONNECTION_ID, they need to change the way they implemented RETIRE_CONNECTION_ID.

Reply:
Yes, I thought that exactly what my proposed sentence point out explicitly. I believe saying this is actually more clear than repeating the text from RFC9000.

Reply:
OK, updating the text.